numba/__init__.py

```diff
@@ -21,7 +21,7 @@
 njit = decorators.njit
 
 # Re export vectorize decorators
-from .npyufunc import vectorize, guvectorize
+from .npyufunc import vectorize, guvectorize, parallel
```

Comment:
On 04/09/15 11:28 AM, Graham Markall wrote:
>
> In numba/__init__.py
> <https://github.com/numba/numba/pull/1404#discussion_r38762997>:
>
> > @@ -21,7 +21,7 @@
> >  njit = decorators.njit
> >  
> >  # Re export vectorize decorators
> > -from .npyufunc import vectorize, guvectorize
> > +from .npyufunc import vectorize, guvectorize, parallel
>
> Is there a reason to re-export the parallel module? (I can't see a
> reason for it)
>

I believe at least one of the tests required it.

-- 

      ...ich hab' noch einen Koffer in Berlin...